src/main/java/com/google/devtools/build/lib/buildtool/BuildRequestOptions.java

```diff
@@ -455,6 +455,17 @@ public boolean useTopLevelTargetsForSymlinks() {
       help = "If this flag is set, replay action out/err on incremental builds.")
   public boolean replayActionOutErr;
 
+  @Option(
+      name = "build_file",
```

Comment:
let's call this different - "build file" is already claimed... maybe "target_file" or "targets_file" or "target_pattern_file"?

Reply:
Went with `target_pattern_file`.